Source/Resource/Plugin/Language/ru.csv

```diff
@@ -236,16 +236,16 @@ BufferSize;Размер буфера
 Cancel;Отмена
 CaptionServiceName;Подпись с именем службы
 CatalogNumber;Номер по каталогу
-Channel(s);Каналы
+Channel(s);Канал(-ы)

```

Comment:
We do it in English, but it is historical, not sure it is the right thing even in English.

Reply:
We do it in English, it is historical, not sure it is the right thing even in English, but if there is no other translation for when we don't know if it is one or more, it may be relevant.
Very subjective.

Reply:
I didn't find any standard, but with a dash looks apparently better. BTW some people use dash, when the ending makes some changes at the end of the word.